src/main/java/proj/kedabra/billsnap/business/repository/AccountBillRepository.java

```diff
@@ -1,15 +1,10 @@
 package proj.kedabra.billsnap.business.repository;
 
-import java.util.stream.Stream;
-
 import org.springframework.data.repository.CrudRepository;
 
-import proj.kedabra.billsnap.business.model.entities.Account;
 import proj.kedabra.billsnap.business.model.entities.AccountBill;
 import proj.kedabra.billsnap.business.model.entities.AccountBillId;
 
 public interface AccountBillRepository extends CrudRepository<AccountBill, AccountBillId> {
```

Comment:
Do we even need this class anymore? There's a codesmell regarding this being injected in `BillServiceImpl` and not being used. I would remove this repository as a whole if we aren't using it anywhere. Otherwise, to remove it at least in the `BillServiceImpl`